lib/data/service/local_storage/local_storage_service.dart

```diff
@@ -0,0 +1,13 @@
+abstract interface class LocalStorageService {
```

Comment:
you should not describe methods of shared prefs in abstraction, you should describe methods regarding your flow